lib/ansible/executor/process/worker.py

```diff
@@ -168,9 +189,23 @@ def _run(self):
         # Set the queue on Display so calls to Display.display are proxied over the queue
         display.set_queue(self._final_q)
 
+        if not self._detached:
+            display.debug(f'Could not detach from stdio: {self._detach_error[1]}')
+            display.error(f'Could not detach from stdio: {self._detach_error[0]}')
+            os._exit(1)
+
         global current_worker
         current_worker = self
 
+        if multiprocessing_context.get_start_method() != 'fork':
+            context.CLIARGS = self._cliargs
+            # Initialize plugin loader after parse, so that the init code can utilize parsed arguments
+            cli_collections_path = context.CLIARGS.get('collections_path') or []
+            if not is_sequence(cli_collections_path):
+                # In some contexts ``collections_path`` is singular
+                cli_collections_path = [cli_collections_path]
+            init_plugin_loader(cli_collections_path)
+
```

Comment:
This code branch effectively does nothing for us currently, since we hardcode the start method.  Although it was required when testing to see if `spawn` works, but requires a changing the hard coded value in `lib/ansible/utils/multiprocessing.py`